server/views/pages/explore.njk

```diff
@@ -23,9 +23,72 @@
     </div>
   </div>
 
+  {# Theme #}
   <div class="row">
     <div class="container">
       <div class="grid">
+        <div class="{{ {s:4, m:8, l:12, xl: 12} | gridClasses }}">
+          <h2 class="heading--large no-v-margin">A drop in the ocean</h2>
```

Comment:
For sure. Perhaps we could have a `shame.css` (or similar). But I'm cool with this as a temporary thing, yeah.